README.md

```diff
@@ -42,6 +42,7 @@ This SDK attempts to follow [semver](https://semver.org/) in a best-effort basis
 
 | React Native SDK | Auth0 SDK |
 | :--------------: | :-------: |
+|     v0.65.0      |  v2.11.0  |
```

Comment:
@Widcket If this is a constraint, then you should update the `package.json` file to reflect that as well.
https://github.com/auth0/react-native-auth0/blob/master/package.json#L29

Reply:
Updated, thanks!